css/style.css

```diff
@@ -6,58 +6,58 @@
 }
 
 .wrapper {
-  margin: 50px auto;
-  width: 280px;
+  -moz-box-shadow:    0px 0px 8px rgba(0,0,0,0.3);
+  -webkit-box-shadow: 0px 0px 8px rgba(0,0,0,0.3);
   background: white;
   border-radius: 10px;
-  -webkit-box-shadow: 0px 0px 8px rgba(0,0,0,0.3);
-  -moz-box-shadow:    0px 0px 8px rgba(0,0,0,0.3);
```

Comment:
@rylnd unfortunately, this isn't *always* true, in some scenarios, some browsers will partially respect both prefixed, and non-prefixed, but only the prefixed actually works (and vice-versa) so if the non-prefixed is last, it could cause things to only partially work. You need to examine this on a case-by-case basis.